plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergColumnHandle.java

```diff
@@ -42,8 +42,7 @@ public class IcebergColumnHandle
     private static final int INSTANCE_SIZE = instanceSize(IcebergColumnHandle.class);
 
     // Iceberg reserved row ids begin at INTEGER.MAX_VALUE and count down. Starting with MIN_VALUE here to avoid conflicts.
-    public static final int TRINO_UPDATE_ROW_ID = Integer.MIN_VALUE;
-    public static final int TRINO_MERGE_ROW_ID = Integer.MIN_VALUE + 1;
+    public static final int TRINO_MERGE_ROW_ID = Integer.MIN_VALUE;
     public static final String TRINO_ROW_ID_NAME = "$row_id";
 
     public static final int TRINO_MERGE_PARTITION_SPEC_ID = Integer.MIN_VALUE + 2;
```

Comment:
We can likely adjust the values (decrease by 1) of the other ID fields now that we don't have anymore Integer.MIN_VALUE + 1